gradle/plugins/src/main/kotlin/com.hedera.gradlebuild.root.gradle.kts

```diff
@@ -14,19 +14,20 @@
  * limitations under the License.
  */
 
-import Utils.Companion.versionTxt
+import com.hedera.gradlebuild.utils.Utils.generateProjectVersionReport
+import com.hedera.gradlebuild.utils.Utils.versionTxt
 import net.swiftzer.semver.SemVer
 
 plugins {
-    id("com.hedera.hashgraph.lifecycle")
-    id("com.hedera.hashgraph.repositories")
-    id("com.hedera.hashgraph.aggregate-reports")
-    id("com.hedera.hashgraph.spotless-conventions")
-    id("com.hedera.hashgraph.spotless-kotlin-conventions")
-    id("com.hedera.hashgraph.dependency-analysis")
+    id("com.hedera.gradlebuild.lifecycle")
+    id("com.hedera.gradlebuild.repositories")
+    id("com.hedera.gradlebuild.aggregate-reports")
+    id("com.hedera.gradlebuild.spotless")
+    id("com.hedera.gradlebuild.spotless-kotlin")
```

Comment:
Do I need to add `com.hedera.gradlebuild.spotless` and `com.hedera.gradlebuild.spotless-kotlin`? I would assume  that `spotless-kotlin` already integrate the other one.

Reply:
:+1: removed.